### PR TITLE
Prefer stateless function validate mhv

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

Part of https://github.com/department-of-veterans-affairs/va.gov-team/issues/8286

This fixes 5 `prefer-stateless-function` linting issues in the validate-mhv-account app

## Testing done

Linting

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/validate-mhv-account/containers/UpgradeMHVAccount.jsx [hunk 1]
```diff
@@ -5,26 +5,24 @@ import MessageTemplate from '../components/MessageTemplate';
...
 import { upgradeMHVAccount } from '../../../platform/user/profile/actions';
 
-class UpgradeMHVAccount extends React.Component {
-  render() {
-    const content = {
-      heading: `Please upgrade your My HealtheVet account`,
-      body: (
-        <>
-          <p>It'll only take us a minute to do this for you. And it's free.</p>
+function UpgradeMHVAccount(props) {
```
Comment (bkjohnson) [inline src/applications/validate-mhv-account/containers/UpgradeMHVAccount.jsx:8]: And here

File: src/applications/validate-mhv-account/containers/ErrorMessage.jsx [hunk 2]
```diff
@@ -11,38 +11,34 @@ import UpgradeAccountFailed from './UpgradeAccountFailed';
...
-    let errorCode = params.errorCode
-      ? params.errorCode.replace(/-/g, '_')
-      : undefined;
-
-    // Capitalize in case of mvi error
-    if (errorCode && errorCode.startsWith('mvi_error')) {
-      errorCode = errorCode.replace('mvi_error_', '').toUpperCase();
-    }
+  // Capitalize in case of mvi error
+  if (errorCode?.startsWith('mvi_error')) {
```
Comment (bkjohnson) [inline src/applications/validate-mhv-account/containers/ErrorMessage.jsx:20]: I converted the `if` to use optional chaining.

**Review by Mottie — APPROVED** [review]
LGTM!

File: src/applications/validate-mhv-account/containers/ErrorMessage.jsx (hunk 2 above)
Comment (jhonnyoddball) [reply src/applications/validate-mhv-account/containers/ErrorMessage.jsx:20]: Good call

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**